sc-tools/sc-server/src/sc-server-impl/sc-memory-json/sc-memory-json-action/sc_memory_handle_link_content_json_action.hpp

```diff
@@ -22,16 +22,18 @@ class ScMemoryHandleLinkContentJsonAction : public ScMemoryJsonAction
 
     auto const & process = [&context, &responsePayload, this](ScMemoryJsonPayload const & atom)
     {
-      std::string const & type = atom["command"];
+      std::string const & type = atom["command"].get<std::string>();
 
       if (type == "set")
         responsePayload.push_back(SetContent(context, atom));
       else if (type == "get")
         responsePayload.push_back(GetContent(context, atom));
       else if (type == "find")
         responsePayload.push_back(FindLinksByContent(context, atom));
-      else if (type == "find_by_substr")
+      else if (type == "find_by_substr" || type == "find_links_by_substr")
         responsePayload.push_back(FindLinksByContentSubstring(context, atom));
+      else if (type == "find_strings_by_substr")
```

Comment:
rename type

Reply:
I propose to coordinate and rename all command names in next releases. I left issue: https://github.com/ostis-ai/sc-machine/issues/177